r128gain/__init__.py

```diff
@@ -265,6 +265,7 @@ def tag(filepath, loudness, peak, *,
       assert(0 <= album_peak <= 1.0)
 
   logger().info("Tagging file '%s'" % (filepath))
+  original_mtime = os.stat(filepath).st_mtime
```

Comment:
Nitpicking: use `os.path.getmtime` instead (same thing, just more readable in my opinion).

Reply:
Absolutely, was'nt aware of that function.